app/src/main/java/ch/epfl/sweng/SDP/game/VotingPageActivity.java

```diff
@@ -197,11 +238,23 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
     protected void onPause() {
         super.onPause();
 
+
+        if (sharingMode == true) {
+            sharingMode = false;
+            return;
+        }
+
+        Log.d("VotingPage","Pausing");
```

Comment:
You can extract "VotingPage" to a private static final String TAG "VotingPageActivity" and use this in all the logs